docs/releases/README.md

```diff
@@ -0,0 +1,41 @@
+# Release Details
+
+This document provides details for Envoy Gateway releases. Envoy Gateway follows the Semantic Versioning [v2.0.0 spec][]
+for release versioning. Since Envoy Gateway is a new project, minor releases are the only defined releases. Envoy
+Gateway maintainers will establish additional release details, e.g. patch releases, at a future date.
+
+## Stable Releases
+
+Stable releases of Envoy Gateway include:
+
+* Minor Releases- A new release branch and corresponding tag are created from the `main` branch. A minor release
+  is supported for 6 months following the release date. As the project matures, Envoy Gateway maintainers will reassess
+  the support timeframe.
+
+Minor releases happen quarterly and follow the schedule below.
+
+## Release Management
+
+Minor releases are handled by a designated Envoy Gateway maintainer. This maintainer is considered the Release Manager
+for the release. The details for creating a release are outlined in the [release guide][].  The Release Manager is
+responsible for coordinating the overall release. This includes identifying issues to be fixed in the release,
+communications with the Envoy Gateway community, and the mechanics of the release.
+
+| Quarter |                        Release Manager                         |
+|:-------:|:--------------------------------------------------------------:|
+| 2022 Q4 |    Daneyon Hansen ([danehans](https://github.com/danehans))    |
+| 2023 Q1 |                              TBD                               |
+
+## Release Schedule
+
+In order to align with the Envoy Proxy [release schedule][], Envoy Gateway releases are produced on a fixed schedule
```

Comment:
WDYT about moving the target release date back another ~week to the last day of each month? That would give us two weeks between Envoy's target date and EG's, which would make it less likely that an Envoy release slip would affect EG's timeline.

Reply:
Not a strong opinion, but: That sounds like a good idea to me.

Reply:
@skriss I'm going to merge since the PR has approvals. I created https://github.com/envoyproxy/gateway/issues/672 to discuss the idea of tracking [envoy-dev](https://hub.docker.com/r/envoyproxy/envoy-dev/tags) to help achieve the 7-day release target, PTAL.